### PR TITLE
Fix for #392 + help enhancement

### PR DESCRIPTION
**Pull request by imran-ansari**
- fix for #392
- dynamic help added to display backend names with their status

### CODE REVIEW

**Comment by imran-ansari** [comment]
PEP8 errors fixed

File: hpedockerplugin/volume_manager.py [hunk 1]
```diff
@@ -1973,6 +1973,7 @@ def _decrypt_password(self, src_bknd, trgt_bknd, backend_name):
         try:
             passphrase = self._etcd.get_backend_key(backend_name)
         except Exception as ex:
+            LOG.error('Exception occurred %s ' % ex)
```
Comment (imran-ansari) [inline hpedockerplugin/volume_manager.py]: As I understand it, if the backend key is not found with ETCD then that's the case of plain credentials. At best, I think this statement should be logged as "INFO" and not "ERROR".

**Comment by wdurairaj** [comment]
One minor thing which needed to be implemented. when we do
` docker volume create -d <alias> -o help` it should also list `-o help=backends` , so the config/create_help.txt requires a change.

Other options is to tell user(s) explicitly in usage.md with the availability `-o help` and `-o help=backends` 
Doc page :  https://github.com/hpe-storage/python-hpedockerplugin/blob/master/docs/usage.md

**Comment by imran-ansari** [comment]
> One minor thing which needed to be implemented. when we do
> ` docker volume create -d <alias> -o help` it should also list `-o help=backends` , so the config/create_help.txt requires a change.
> 
> Other options is to tell user(s) explicitly in usage.md with the availability `-o help` and `-o help=backends`
> Doc page : https://github.com/hpe-storage/python-hpedockerplugin/blob/master/docs/usage.md

Thanks William! Addressed the review comment.

**Pull request merged**